backend/plugins/frontline_api/src/modules/integrations/call/utils.ts

```
@@ -0,0 +1,957 @@
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        request: {
+          action: 'challenge',
+          user: apiUser,
+          version: '1.0.20.23',
+        },
+      }),
+    });
+
+    const data = await challengeResponse.json();
+    const { challenge } = data?.response;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)